adobe_vipm/adobe/mixins/order.py

```
@@ -186,15 +191,14 @@ def create_preview_order(self, context: Context) -> dict | None:
-        deployment_id = get_deployment_id(context.order)
-        if deployment_id:
-            for line_item in payload["lineItems"]:
-                line_item["deploymentId"] = deployment_id
-                line_item["currencyCode"] = authorization.currency
-        else:
-            payload["currencyCode"] = authorization.currency
+        if context.upsize_lines:
+            try:
+                self._process_upsize_lines(context, flex_discounts, payload)
+            except ProcessingUpsizeLinesError as error:
+                raise AdobeCreatePreviewError(error)
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)